ProgramowanieBot/ProgramowanieBot.csproj

```diff
@@ -8,6 +8,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.5">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.5" />
```

Comment:
Where is it used?

Reply:
https://github.com/Programowanie-Discord/ProgramowanieBot/pull/3#discussion_r1174467469